es-vue-base/src/lib-components/EsFileUpload.vue

```diff
@@ -0,0 +1,230 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
```

Comment:
I made it so that clicking on the div instead of just the button will open the file picker. This is due to the fact the cursor changes to make the div look clickable, and I figured the user would want something to happen when they see that the div is a button! If that is misguided let me know and I can attempt a different workaround

Reply:
Not an issue at all just needed those attributes on the div since it is now "focusable"